src/actions/app.js

```diff
@@ -117,11 +117,6 @@ export const { hideAlert } = createActions({
   [ActionTypes.HIDE_ALERT]: index => index,
 });
 
-export const { getConfiguration } = createActions({
-  [ActionTypes.GET_CONFIGURATION_SUCCESS]: configuration => configuration,
-  [ActionTypes.GET_CONFIGURATION]: configuration => configuration,
```

Comment:
I think this is the only place we're using these action types, might as well remove them from the enum?

Reply:
We are using the SUCCESS, but not the regular one so I removed it.